### PR TITLE
Use logging rather than stderr or stdout output when it makes sense.

### PR DESCRIPTION
**Pull request by bcarlin**
Stderr is used when the service cannot start. If it can, there is no reason not to log.
This PR is just housekeeping.

To do later (because it might break some workflows, even if [xkcd #1172](https://xkcd.com/1172/)): use warning only for real warnings not to create false positive in monitoring systems.

### CODE REVIEW

**Comment by fredericBregier** [comment]
For me, fine !

I just have one proposition in mind: what about change logger.debug to logger.info ? The idea of those system.out was to inform clearly what happens during the creation / upgrade process, such that someone could send us the log. However, by default LOG level is set to WARN such that they will not be written... So perhaps up to logger.warn...
The idea is that this kind of actions are quite rare so logging in Warn mode is not that bad, since it could explains later on some issues...

WDYT ?

**Comment by bcarlin** [comment]
_Tl;dr: This PR is too supperficial and not really well thought. I close it and work on a better "output" normalisation solution_ 

For me, it is a matter of two things : 
* What meaning do we give to each level
* What does the operator of the application expects.

On the first question, my view is very literal : a warning is a potential problem, info is just informative (but no action is required).
If I take the doc of slf4j, the [suggested sematics](http://www.slf4j.org/apidocs/index.html) for 
> WARN
>      The WARN level designates potentially harmful situations.
> INFO 
>      The INFO level designates informational messages that highlight the progress of the application at coarse-grained level.
> DEBUG
>      The DEBUG Level designates fine-grained informational events that are most useful to debug an application.

So maybe OK, I will up the level of SQL query to INFO. But WARN seems to me far-fetched.

On this not, some operators asked me to drop the level of some messages to info because they did not understand why messages like "The server started successfully" was a warning.

The more I Think about it, the more I see there is a bigger normalization work to do.
For example in this block (https://github.com/waarp/WaarpGatewayFtp/blob/master/src/main/java/org/waarp/gateway/ftp/config/FileBasedConfiguration.java#L858) :

```java
    if (value != null && (!value.isEmpty())) {
            snmpConfig = value.getString();
            logger.warn("SNMP configuration file: " + snmpConfig);
            File snmpfile = new File(snmpConfig);
            if (snmpfile.canRead()) {
                if (!SnmpConfiguration.setConfigurationFromXml(snmpfile)) {
                    logger.warn("Bad SNMP configuration file: " + snmpConfig);
                    snmpConfig = null;
                }
            } else {
                logger.warn("Cannot read SNMP configuration file: " + snmpConfig);
                snmpConfig = null;
            }
        } else {
            logger.warn("NO SNMP configuration file");
        }
```

WARN is used for 2 different things :
* informative messages ("this is the config file" and "you don't wan't to use SNMP")
* errors ("the configuration file cannot be read" and "the xml file is malformed") which disable completely SNMP monitoring

With one single intent (correct me if I'm wrong) : we want to see these messages even if the log level is set too low application-wide for them to be logged.

So I'm closing this PR and I will come back later with a better an more complete solution !

**Comment by fredericBregier** [comment]
I understand. I'm also in the middle between INFO and WARN often.
What I feel, probably wrongly, is:
- WARN is the default level in production since INFO might be verbose (not as DEBUG, but however lot more than WARN)
- WARN is obviously to warn about something, not necesseraly an error (or it should be ERROR), but to warn someone that something happens
  - In the example of SNMP, WARN is not an error but a real info that SNMP will not be started since it cannot, but is not a blocking state to continue. So I feel like not an INFO, but a real WARN.
- INFO is more general information, not as DEBUG already, but in the middle, probably misused
- ERROR most of the time produces a stop of the action (since we do not have FATAL). So ERROR means something really wrong happens. For a transfer, for instance, the transfer is blocked and in error status, while a retry is only a Warning (before a real Error if the 3 retries are over).

In the context of production, I often like to have the following information when I start a service:
- it really started
- it stopped
- it is correctly configured and what are the main configuration items (for instance the listening port)
- when I do special administration tasks (like upgrading the database), I like to have most of the information needed. Perhaps the best would be to set the command using level INFO, such that INFO is going to be logged. But if not (like almost everyone is using the very same logger.conf so WARN level), I will miss this very important log in my production flow.

So the reason the code is sometime using WARN where it could be INFO, but in order that "production supervision" gets the important information (start/stop/upgrade/...).

This does not mean that all messages are of the same interest, so you could freely decide which one to move from WARN to INFO, keeping in mind that some shall be kept in WARN mode.

**Comment by fredericBregier** [comment]
I've an idea that could allow everyone to set the level it wants (if not going through different logger.conf level specification as WARN to INFO and reverting):
- adding an extra option (through command line as -Dwaarp.warn.production or something) that will choose between WARN or INFO for those messages where we are not sure about the right level but still considering it could be a good idea to have those in the log in production situation, by default being INFO.

WDYT?
